selfservice/hook/verification.go

```diff
@@ -37,19 +39,19 @@ func NewVerifier(r verifierDependencies) *Verifier {
 	return &Verifier{r: r}
 }
 
-func (e *Verifier) ExecutePostRegistrationPostPersistHook(_ http.ResponseWriter, r *http.Request, f *registration.Flow, s *session.Session) error {
+func (e *Verifier) ExecutePostRegistrationPostPersistHook(w http.ResponseWriter, r *http.Request, f *registration.Flow, s *session.Session, continueWith *flow.ContinueWithContainer) error {
```

Comment:
I don't like this function parameter. It is used to carry a slice of `ContinueWith` items, which the hook executors can add their actions (`verification_ui` or `set_token`) to, which is then used to return/write into the response.

If you have another idea, please LMK.

Reply:
The registration flow is not returned after the registration is completed. We return an `registration.APIFlowResponse` which contains session, identity, a token, and now this.

I guess I could add it to the flow and then just carry it over, but we don't want to persist this slice. So this would then be some field that has some meaning sometimes, which just feels wrong to me.

Reply:
I refactored it, to use the flow as the "carrier" for these items. Not super happy with it, but a lot better. Great suggestion!